custom/icds_reports/static/js/angular-services/baseControllers.service.js

```diff
@@ -15,6 +15,10 @@ window.angular.module('icdsApp').factory('baseControllersService', function() {
                 $location.search('location_name', loc.name);
             }
         };
+        vm.addAdditionalFilters = function (gender, age) {
+            $location.search('gender', gender);
+            $location.search('age', age);
```

Comment:
nit: I think it'd be slightly nicer if it only added these if necessary. either based on whether values exist or the report is setup to include them. also if you did that it might be good to rename the method to `addAdditionalFiltersIfNecessary`

Reply:
I'm not sure of the reason. But this code already does the same job. For reports where no filters should be shown, gender and age is removed from url.

Reply:
Great! :man_shrugging: